centreon/src/Core/HostGroup/Infrastructure/API/DuplicateHostGroups/DuplicateHostGroupsController.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/*
+ * Copyright 2005 - 2025 Centreon (https://www.centreon.com/)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * For more information : contact@centreon.com
+ *
+ */
+
+declare(strict_types=1);
+
+namespace Core\HostGroup\Infrastructure\API\DuplicateHostGroups;
+
+use Centreon\Application\Controller\AbstractController;
+use Core\HostGroup\Application\UseCase\DuplicateHostGroups\DuplicateHostGroups;
+use Core\Infrastructure\Common\Api\StandardPresenter;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\Attribute\MapRequestPayload;
+use Symfony\Component\Security\Http\Attribute\IsGranted;
+
+#[IsGranted(
+    'hostgroup_duplicate',
```

Comment:
```suggestion
    HostGroupVoters:: HOSTGROUP_DUPLICATE,
```

(and add HostGroupVoters into use), i have done it too on the enable/disable, I will talk quickly about it in dev sync, but I think it should be a good practice to use the Voters Constant here to avoid errors if for some reason the value change somewhere.